ff/smirnoff_1.1.0.py

```diff
@@ -343,15 +343,17 @@
   # hand generated
   'GBSA': {
     'params': [
-      ['[*:1]', .217294, 0.16865],
-      ['[#1:1]', .179923, 1.00254],
-      ['[#6:1]', .209618, 0.93122],
-      ['[#7:1]', .172778, 0.77446],
-      ['[#8:1]', .227665, 0.88298],
-      ['[#9:1]', .164722, 0.80448],
-      ['[#14:1]', .182741, 0.94334],
-      ['[#15:1]', .067724, 0.83768],
-      ['[#16:1]', .220727, 0.7523]
+       ['[*:1]',  0.15,  0.8],
+       ['[#1:1]', 0.12,  0.85],
+       ['[#1:1]~[#7]', 0.13,  0.85],
+       ['[#6:1]', 0.17,  0.72],
+       ['[#7:1]', 0.155,  0.79],
+       ['[#8:1]', 0.15,  0.85],
+       ['[#9:1]', 0.15,  0.88],
+       ['[#14:1]', 0.21,  0.8],
+       ['[#15:1]', 0.185,  0.86],
+       ['[#16:1]', 0.18,  0.96],
+       ['[#17:1]', 0.17,  0.8],
     ],
```

Comment:
These all match with the "really old" implementation in Amber (located in $AMBERHOME//AmberTools/src/leap/src/leap/unitio.c). 
In addition, the openMM implementation makes some stupid mistakes: H-N hydrogen GB radii is mistakenly switched with H-C hydrogen GB radii, etc. I am probably wrong, but they are really messed up in openmm/wrappers/python/simtk/openmm/app/data/amber*_obc.xml

Reply:
Any chance you can open an issue to point out these problems?
We need to revisit how we apply GBSA parameters anyway!

http://github.com/openmm/openmm/issues

Reply:
Yeah I'll do that!